backend/typescript/models/testSession.model.ts

```diff
@@ -0,0 +1,81 @@
+import mongoose, { Schema, Document } from "mongoose";
+
+/**
+ * This interface holds information about the result of a single student
+ * on a test
+ */
+export interface Result {
+  /** the name of the student */
+  student: string;
+  /** the score of the student */
+  score: number;
+  /**
+   * a list corresponding to the question list with each fielding indicating
+   * whether the student got the question right or not
+   */
+  breakdown: [boolean];
+}
+
+const ResultSchema: Schema = new Schema({
+  student: {
+    type: String,
+    required: true,
+  },
+  score: {
+    type: Number,
+    required: true,
+  },
+  breakdown: {
+    type: [Boolean],
+    required: true,
+  },
+});
+
+/**
+ * This document contains information about a single test session
+ */
+export interface TestSession extends Document {
+  /** the unique identifier of the test session */
+  id: string;
+  /** the ID of the corresponding test from the Test collection */
+  test: string;
+  /** the ID of the teacher administering the test from the User collection */
+  teacher: string;
+  /** the ID of the school that's administering the test from the School collection */
+  school: string;
+  /** the grade level that is being tested */
+  grade_level: number;
+  /**
```

Comment:
Let's also add a field for `access_code` (the access code that students can use to access the test when it is live) and `start_time` (when test is started). Sorry this wasn't included in the scope of this ticket prior to you working on it, just thought of this!